Tweaking FoodAdvisor Readme File

malgamves opened the pull request:
I made the instalation instructions a little more explicit. Inside the ./api folder is another ./api folder and the current state of the readme directs user to cd into ./api/api which can cause some installation errors

lauriejim commented on README.md:
````diff
@@ -10,28 +10,30 @@ Clone FoodAdvisor
 ```
 
-## 2. Start Strapi from the ./api folder
+Navigate to your *./foodadvisor* folder by running `cd foodadvisor` from your command line.
````
```diff
- Navigate to your *./foodadvisor* folder by running `cd foodadvisor` from your command line.
+ Navigate to your `./my-projects/foodadvisor` folder by running `cd foodadvisor` from your command line.
```

I don't think that *./foodadvisor* is the right path to indicate due to the catch that we mention in **step 1** `./my-projects` path using `./`.
So the usage of `./` for `./foodadvisor` is a bit confusing.

Second point, is that you used *italic* style for a path and after the `code` style is used.
I think we should use `code` here.

malgamves replied:
Thanks for the catch, will fix it.

lauriejim commented on README.md:
```diff
@@ -10,28 +10,30 @@ Clone FoodAdvisor
+## 2. Start Strapi
 
-Run the following from your command line:
+Naviagte to your *./api folder* by running `cd api` from your command line.
```
I think it's the same here for `./` and `code` style

lauriejim commented on README.md:
````diff
@@ -10,28 +10,30 @@ Clone FoodAdvisor
+yarn && yarn run seed && yarn develop
 ```
 
 Then, follow [Step 3. Create an admin user](https://strapi.io/documentation/3.0.0-beta.x/getting-started/quick-start-tutorial.html#_3-create-an-admin-user) to create an `Administrator`.
````
I know that it's not an update you did, but can you please update the link here to redirect to the last version of the docs instead of the beta.
Worth it to mention that it's also **step 2** instead of **step 3**.
Here is the doc link for the stable: https://strapi.io/documentation/v3.x/getting-started/quick-start.html#_2-create-an-administrator-user

lauriejim commented on README.md:
```diff
@@ -10,28 +10,30 @@ Clone FoodAdvisor
+## 3. Start the front-end 
 
-`Path: ./my-projects/foodadvisor/client`:
+Naviagte to your *./client folder* by running `cd client` from your command line.
```
I think it's the same here for `./` and `code` style

lauriejim commented on README.md:
````diff
@@ -10,28 +10,30 @@ Clone FoodAdvisor
 
 ```
-cd client && yarn install && yarn start
+yarn install && yarn start
````
```diff
- yarn install && yarn start
+ yarn && yarn start
```

Can you also please remove the `install` of `yarn install` to match the same command we used in the Strapi setup step.

malgamves replied:
on it 👍

lauriejim reviewed (changes requested):
Hello @malgamves , I suggested some updates.
I have an overall question, did we have some issues that required this update?

Ping @meganelacheny, in the future, I think a review of readme could be done.
Not a prio at all. This message is just a note/idea/mention.

malgamves commented:
Hey @lauriejim - thanks for the comments. Was installing it and had some issues so thought why not fix them ;) 
Will def ping Megane next time. 

I've added the fixes you requested and fixed the DCO issue. Please have a look.

lauriejim reviewed (approved):
LGTM
